src/test/java/io/crate/testing/CrateTestServerTests.java

```diff
@@ -49,53 +48,53 @@ private CrateTestServer serverOf(String version) {
                 "localhost",
                 "1.1.1.1");
     }
-
+    
     @Test
     public void testPrepareSettingsForAnyVersion() {
         CrateTestServer server = serverOf("0.0.0");
         Map<String, Object> settings = server.prepareSettings();
-
+        
         assertThat(settings.get("cluster.name"), is("test-cluster"));
         assertThat(settings.get("network.host"), is("localhost"));
         assertThat(settings.get("http.port"), is(4200));
         assertThat(settings.get("transport.tcp.port"), is(4300));
         assertThat(settings.get("psql.port"), is(5432));
         assertThat(settings.get("psql.enabled"), is(true));
     }
-
+    
     @Test
     public void testPrepareSettingsForLt2_0() {
         CrateTestServer server = serverOf("1.0.0");
         Map<String, Object> settings = server.prepareSettings();
-
+        
         assertThat(settings.get("discovery.zen.ping.multicast.enabled"), is("false"));
         assertThat(settings.get("index.storage.type"), is("memory"));
     }
-
+    
     @Test
     public void testPrepareSettingsForLt4_0() {
         CrateTestServer server = serverOf("3.0.0");
         Map<String, Object> settings = server.prepareSettings();
-
+        
         assertThat(settings.get("discovery.zen.ping.unicast.hosts"), is("localhost,1.1.1.1"));
     }
-
+    
     @Test
     public void testPrepareSettingsForGte4_0() {
         CrateTestServer server = serverOf("4.0.0");
         Map<String, Object> settings = server.prepareSettings();
-
+        
         assertThat(settings.get("discovery.seed_hosts"), is("localhost,1.1.1.1"));
         assertThat(settings.get("cluster.initial_master_nodes"), is("localhost,1.1.1.1"));
     }
-
+    
     @Test
     public void testJDK8IsUsedForCrateLt3_2() {
         HashMap<String, String> env = new HashMap<>();
         CrateTestServer.prepareEnvironment(env, "3.0.0");
-        assertThat(env.get("JAVA_HOME"), containsString("1.8"));
+        assertThat(env.get("JAVA_HOME"), anyOf(containsString("1.8"), containsString("java-8")));
     }
-
+    
```

Comment:
Could you please remove all the added white spaces inside this file?

Reply:
It has been fixed.